File: proj/pollinone/app/src/main/java/com/cs442/sexysuckzoo/pollinone/MainActivity.java
package com.cs442.sexysuckzoo.pollinone;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    // button callback
    public void joiningPoll(View v) {
        Intent intent = new Intent(this, SearchingPoll.class);
        startActivity(intent);
    }

    // button callback
    public void creatingPoll(View v) {
        Intent intent = new Intent(this, CreatingPoll.class);
        startActivity(intent);
    }
}

File: proj/pollinone/app/src/main/java/com/cs442/sexysuckzoo/pollinone/MainActivity.kt
package com.cs442.sexysuckzoo.pollinone

import android.content.Intent
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.util.Log
import android.view.View

import com.github.kittinunf.fuel.core.FuelManager

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        FuelManager.instance.basePath = applicationContext.resources.getString(R.string.api_root)
        Log.wtf("basepath", FuelManager.instance.basePath)
    }

    // button callback
    fun joiningPoll(v: View) {
        val intent = Intent(this, JoiningPoll::class.java)
        startActivity(intent)
    }

    // button callback
    fun creatingPoll(v: View) {
        val intent = Intent(this, CreatingPoll::class.java)
        startActivity(intent)
    }
}
